File: Skillrack/C/starter/absolute difference.c
#include <stdio.h>
#include <stdlib.h> // For abs() function

int main() {
    int a, b; // Declare two integer variables

    // Accept input for two integers
    printf("Enter two integers: ");
    scanf("%d %d", &a, &b); // Read integers from user input

    // Print the absolute difference
    printf("Absolute difference: %d\n", abs(a - b)); // Calculate and print absolute difference

    return 0; // Indicate successful completion
}

File: Skillrack/java/starter/absolute difference.java
import java.util.Scanner;

public class Hello {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1 = sc.nextInt();  // Read first number
        System.out.print("Enter the second number: ");
        int num2 = sc.nextInt();  // Read second number
        
        // Calculate the absolute difference
        int difference = Math.abs(num1 - num2);
        
        // Print the result
        System.out.printf("The absolute difference is: %d%n", difference);
        
        sc.close();  // Close the scanner
    }
}
